front/src/modules/simulationResult/components/SpaceTimeChart/useGetProjectedTrainOperationalPoints.ts

```diff
@@ -40,13 +51,51 @@ const useGetProjectedTrainOperationalPoints = (
         };
         const { operational_points } = await postPathProperties(pathPropertiesParams).unwrap();
```

Comment:
i think you can optimize operational_points set doing 

```
const { operational_points } = await postPathProperties({
        infraId,
        props: ['operational_points'],
      }).unwrap();
```

Reply:
Hmm we need also `pathPropertiesInput` for the request. I think it's more clearer to leave the object before and pass it to the endpoint. Is that for you ?

Reply:
Yes sorry the code was wrapped but still, if pathPropertiesParams is not to be reused, it's not useful. You should just put args in the request

Reply:
Ok done :)